feature-flag/src/main/kotlin/com/linecorp/android/featureflag/FeatureFlagJavaFileWriter.kt

```diff
@@ -72,14 +72,21 @@ internal class FeatureFlagJavaFileWriter(
         // static analyzer.
         val actualFlagValue = flag.value.toJavaValue(isRequiredLiteralValue)
 
+        if (flag.hasOption(FeatureFlagOption.DEPRECATED)) {
+            emitAnnotation("Deprecated")
```

Comment:
I think `emitAnnotation(Class<? extends Annotation> annotationType)` is a little bit more strict and safer.

Reply:
Done